config/default.json

```diff
@@ -44,7 +44,9 @@
   "mapproxy": {
     "mapproxyApiUrl": "http://localhost:8086",
     "mapproxyYamlDir": "/mapproxy/mapproxy.yaml",
-    "seedYamlDir": "/mapproxy/seed.yaml"
+    "seedYamlDir": "/mapproxy/seed.yaml",
+    "geometryTxtFile": "/mapproxy/coverage.json",
+    "seedProgressFileDir": "/mapproxy/mapproxy_seed_progress"
```

Comment:
It's internal config for the default generated file (binary) and should be configured as part of the deployment.
if on the future we would like to change /mapproxy as workdir it will be possible insert deployment config (custom variable + helm)